hypervisor/src/cpu.rs

```diff
@@ -477,4 +481,13 @@ pub trait Vcpu: Send + Sync {
     fn set_tsc_khz(&self, _freq: u32) -> Result<()> {
         Ok(())
     }
+    fn get_cpuid_values(
```

Comment:
This should be behind `x86_64` feature gate.

Reply:
Also, why you squashed the commit `hypervisor, vmm: Add API to retrieve cpuid leaf ` into this commit? It should be a separate commit and should contains changes only from the `hypervisor` crate.

Reply:
The `hypervisor, vmm: Add igvm module and loader module` commit needs a cleanup for its commit message.